opencart2.0.x-2.2.x/upload/catalog/controller/payment/xendit.php

```diff
@@ -168,6 +182,7 @@ private function process_order($response, $order_id)
             $this->response->setOutput($message);
         } else {
             $message = 'Invoice not paid or settled. Cancelling order. Charge id: ' . $response['id'];
```

Comment:
Please change text to: 'Invoice not paid or settled. Cancelling order. **Invoice ID**: '